File: v4/lua/long_text/main.lua
local json = require('cjson')
local hc = require('lib.httpclient').new()

local url = 'http://api-text-bj.fengkongcloud.com/longtext/v4'
local accessKey = '{ACCESS_KEY}'

local payload = json.encode({
  accessKey = accessKey,
  appId = 'default',
  eventId = 'default',
  type = 'ALL',
  data = {
    text = '{TEXT}',
    tokenId = '{UID}',
  },
})

local res = hc:post(url, payload)
local data = json.decode(res['body'])
-- 通过 code 和 riskLevel 判断返回，具体请参考接口文档
print(res['body'])

File: v4/python/long_text/main.py
import requests

url = 'http://api-text-bj.fengkongcloud.com/longtext/v4'
access_key = '4Ky6AV4hE0pWLeG1bXNw'
text = '{TEXT}'
uid = '{UID}'

payload = {
    'accessKey': access_key,
    'appId': 'default',
    'eventId': 'default',
    'type': 'ALL',
    'data': {
        'text': text,
        'tokenId': uid,
    }
}

res = requests.post(url, json=payload)
# 通过 code 和 riskLevel 判断返回，具体请参考接口文档
if res:
    print(res.json())
